From 71650b18b78f52e96512a7695a0d39762c1ee408 Mon Sep 17 00:00:00 2001
From: Alex Marginean <alexandru.marginean@nxp.com>
Date: Mon, 6 Jan 2020 16:35:08 +0200
Subject: [PATCH] scripts: Makefile: Enable creation of _symbols_ DT node for
 overlays

DT compiler supports -@ flag which indicates that _symbols_ node should be
automatically created during compilation.  This node is required for
using DT overlays.  The generated DTB is backward compatible, the _symbols_
node can be safely ignored if overlays are not used.

Signed-off-by: Alex Marginean <alexandru.marginean@nxp.com>
---
 scripts/Makefile.lib | 3 +++
 1 file changed, 3 insertions(+)

--- a/scripts/Makefile.lib
+++ b/scripts/Makefile.lib
@@ -255,6 +255,9 @@ endif
 
 DTC_FLAGS += $(DTC_FLAGS_$(basetarget))
 
+# Enable creation of _symbols_ node for DT overlays
+DTC_FLAGS += -@
+
 # Generate an assembly file to wrap the output of the device tree compiler
 quiet_cmd_dt_S_dtb= DTB     $@
 cmd_dt_S_dtb=						\
